docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  db:
+    image: postgres:16
```

Comment:
Not sure if we get away without any geo-stuff in this service (which would require a postgis image), but that easily changeable so we can leave it like this for the moment.

Reply:
OK, as long as the data is compatible I guess?